src/constants.cpp

```diff
@@ -2975,6 +3009,18 @@ const char *pc_race_types[] =
     "Night-One",
     "Dryad",
     "Dragon",
+    "Dragon",
+    "Serpent",
+    "Drake",
```

Comment:
Do these need to be identified as 'Drake (Human)' or 'Humanoid Drake' or something? We won't be able to differentiate them as staff if they're all the same string.

Reply:
Bastardized em so they fit